app/responsive-demo/page.tsx

```
@@ -0,0 +1,474 @@
+                <CardHeader>
+                  <CardTitle>Advanced Visibility with Breakpoints</CardTitle>
+                  <CardDescription>
+                    More precise control with the Show component that accepts from/until props.
+                  </CardDescription>
+                </CardHeader>
+                <CardContent className="space-y-4">
+                  <Show from="sm" until="lg">
+                    <div className="bg-pink-100 dark:bg-pink-900 p-4 rounded">
+                      <p className="font-medium">This is only visible on tablets</p>
+                      <p className="text-sm text-muted-foreground">
+                        Using Show component with from="sm" until="lg"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)